test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua

```diff
@@ -0,0 +1,129 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PTU with “disallowed_by_external_consent_entities_off” struct
+--
+-- Description:
+-- In case:
+-- SDL receives PolicyTableUpdate with “disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]” -> of "<functional grouping>"
+-- -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PTU as valid (with the pre-conditions of all other valid PTU content)
+-- b. add this “disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]” field with the corresponding value
+-- to the corresponding "<functional grouping>" to the Policies database.
+--
+-- Preconditions:
+-- 1. Start SDL (make sure 'disallowed_by_external_consent_entities_off' section is omitted in PreloadedPT)
```

Comment:
@dboltovskyi, please add step to check that 'disallowed_by_external_consent_entities_off' section is omitted in PreloadedPT